### ISSUE TITLE
Add the possibility to upload the Excel file template instead of the Input data DB

### ISSUE THREAD

**Issue by e-zaline** [irena-flextool/flextool-web-interface#19]

**Comment by soininen** [comment]
This has now been implemented and merged to the `main` branch.

**Comment by e-zaline** [comment]
Thanks @soininen !
May I suggest to make it more explicit that the user can import either a DB or the Excel? I saw that it is part of the tooltip of the button, but I would suggest to write it in the text:
Example:
Import an existing project from a database or an Excel template.
Download model database here.
Download model Excel template here. (+link)

**Comment by soininen** [comment]
Fair enough, the text could indeed be improved and and an option to download the Excel template added.
